src/main/java/com/money_account_service/entities/TransferEntity.java

```diff
@@ -0,0 +1,40 @@
+package com.money_account_service.entities;
+
+import jakarta.persistence.*;
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.Instant;
+
+@Table(name = "transfers")
+@Entity
+@Builder
+@Data
+public class TransferEntity {
```

Comment:
You use two words interchangeably: transfer & transaction. We should decide and commit to just one.